### DISCUSSION TITLE
Implementing SRI hashes for Alpine via CDN

### DISCUSSION THREAD

**Discussion by HeartthrobRob** [alpinejs/alpine#4293] [4. General]
Have the dev team considered updating the CDN link on the docs to include a server-side SRI integrity hash? These are generated from sites like: https://www.srihash.org/ and protect against server-side malicious activity. 

Rationale documented here (specifically last paragraph): https://blog.wesleyac.com/posts/why-not-javascript-cdn

**Comment by ekwoka** [comment]
Well, the main showed links are not version bound links, so they can't be hashed.

there is just the one that could use them.

But the docs are in this Repo. So you can make the PR to add it to that spot. :)
